duckduckgo/iOS
# swipe to close tabs

Task/Issue URL: https://app.asana.com/0/392891325557410/1152162761690392
Tech Design URL:
CC:

**Description**:

Allows users to swipe to close tabs.

**Steps to test this PR**:
1. Open some tabs.
1. Swipe to close, top one, middle one, bottom one, last one, etc.
1. Ensure tabs close 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
